documentation/docs/03-loading.md

```diff
@@ -54,7 +54,7 @@ type LoadOutput = {
 </script>
 ```
 
-`load` is the SvelteKit equivalent of `getStaticProps` or `getServerSideProps` in Next.js or `asyncData` in Nuxt.js.
+`load` is similar to Next.js's `getStaticProps` or `getServerSideProps` or Nuxt.js's `asyncData`, except that it runs on both the server and the client.
```

Comment:
does `asyncData` only run on the server? I think it runs in both, which would make this sentence confusing. We could either clarify that, or drop the Nuxt mention altogether

```suggestion
`load` is similar to `getStaticProps` or `getServerSideProps` in Next.js, except that it runs on both the server and the client.
```